api/protobuf/core/api_container_service.proto

```diff
@@ -78,6 +78,9 @@ message Port {
   string maybe_application_protocol = 3;
   // The wait timeout duration in string
   string maybe_wait_timeout = 4;
+
+  // this is useful in the cloud context; ignore in local
+  optional bool locked = 5;
```

Comment:
@imjuangarcia new backend features that you will be using

Reply:
this will flow down to the response of GetServices in the frontend where the ports information exists, it will be optional, if it is not set then don't show anything on the ui if it is false then unlock if it is true then locked